lib/otp/views/configure_chalresp_dialog.dart

```diff
@@ -31,6 +31,7 @@ import '../../widgets/app_input_decoration.dart';
 import '../../widgets/app_text_field.dart';
 import '../../widgets/responsive_dialog.dart';
 import '../keys.dart' as keys;
+import '../keys.dart';
```

Comment:
remove this and use the prefix (instead of `generateSecretKey` use `keys.generateSecretKey` and so on)